Changelog.md

```diff
@@ -25,6 +25,8 @@ Compiler Features:
  * Build system: Switch the emscripten build from the fastcomp backend to the upstream backend.
  * Code Generator: Do not introduce new internal source references for small compiler routines.
  * Commandline Interface: Adds new option ``--base-path PATH`` to use the given path as the root of the source tree (defaults to the root of the filesystem).
+ * Commandline Interface: Exit with an error if more than one mode (i.e. ``--assemble``, ``--strict-assembly``, ``--yul``, ``--link``, ``--standard-json`` or ``--import-ast``) is selected at the same time.
```

Comment:
Please move.

Reply:
Probably this is also too detailed. "Prevent some incompatible commandline options to be used together".

Reply:
Done.